### ISSUE TITLE
Missing successful global messages when an additional Payment Method or a Shipping Address is added during checkout

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13699]
**Describe the bug**
- During the checkout process there is no successful global message when an additional Payment Method or a Shipping Address is added.
- Screen reader users wouldn't be able to tell if the additional Payment Method or Shipping Address was in fact added.
- Since this is likely an issue of a missing global message, it could be fixed by including one and adding ARIA attributes as suggested in this ticket #13677.

**Tell us the version of Spartacus**
- Library version: 4.0.0.

**To Reproduce**
1. Register a new user.
2. Add items to cart and complete checkout with one Payment Method and one Shipping Address.
3. Open any Screen Reader (VO, NVDA, JAWS).
4. Start a new order and during checkout add another Payment Method and another Shipping Address.
5. Hear absence of confirmation about anything being added.

**Expected behavior**
- Screen Reader users hear a confirmation when they add an additional Payment Method or Shipping Address during checkout.

**Screenshots**

![Screen Shot 2021-09-13 at 6 41 30 PM](https://user-images.githubusercontent.com/15113219/133165806-6762a037-e8a0-48ed-981e-d994536af0cc.png)
![Screen Shot 2021-09-13 at 6 41 40 PM](https://user-images.githubusercontent.com/15113219/133165807-183340fd-4a77-44dc-956b-9d9c51c92812.png)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.
![checkoutAddress](https://user-images.githubusercontent.com/45409515/146214299-d8803501-f234-4987-82a6-57a0f375c70f.png)
![checkoutPayment](https://user-images.githubusercontent.com/45409515/146214302-1f228b44-d5bf-4b7f-908e-d89f5de40722.png)
